guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -12,8 +12,17 @@ endif::[]
 ifdef::satellite[]
 * Review Known Issues before you begin an upgrade.
 For more information, see {ReleaseNotesURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].
+* If you run {Project} in a disconnected environment, the following additional prerequisites must be met:
+  - You must obtain and deploy Leapp metadata manually. For more information, see https://access.redhat.com/articles/3664871[Leapp utility metadata in-place upgrades of RHEL for disconnected upgrades].
+  - You require access to {RHEL} and {Project} packages.
+  - Follow the steps for obtaining ISO files for {RHEL} 8 and {Project} in xref:upgrading_a_disconnected_satellite[].
+* For more information on customizing the Leapp upgrade for your environment, see https://access.redhat.com/articles/4977891[Customizing your {RHEL} in-place upgrade].
+* When running `leapp preupgrade` and `leapp upgrade`, you must pass `--enablerepo <id>` for each manually defined repository from the previous step.
```

Comment:
```suggestion
```

I think it's sufficient to be noted in the actual steps, it's not a pre-req.

Reply:
@evgeni I moved it right after the step talking about running in a disconnected environment. Let me know if that's a good spot for it.